docs/04-operation-guides/security/all-02-authentication-in-kyma.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Authentication in Kyma
+---
+
+## User authentication
+
+The identity federation in Kyma is managed through an [OpenID Connect (OIDC)](https://openid.net/connect/)-compliant identity provider.
+
+The diagram shows the user authentication flow:
+
+![Authentication diagram](./assets/all-authentication-diagram.svg)
+
+1. Access Kyma Dashboard.
+2. Kyma Dashboard redirects you to an OIDC-compliant identity provider to handle the authentication.
+3. After successful authentication, the identity provider issues a JWT token that is stored in the browser session and used for all subsequent requests.
+
+### ID Tokens
+
+ ID Tokens are JSON Web Tokens (JWTs) signed by an OIDC-compliant identity provider and returned as part of the OAuth2 response that attest to the end user's identity.
+ An example decoded JWT looks as follows:
+
+```json
+{
+  "iss": "http://127.0.0.1:5556/dex",
```

Comment:
sample issuear should not refer to dex

Reply:
Thx, I changed it to the example you provided.